Dockerfile

```
@@ -28,12 +28,7 @@ RUN git clone https://github.com/mendersoftware/mender.git . && \
     git checkout $MENDER_CLIENT_VERSION && \
     DESTDIR=/install-modules-gen make install-modules-gen
 
-FROM docker:25-dind
-ARG MENDER_APP_UPDATE_MODULE_VERSION
+FROM debian:12.5
```

Please use the slim version
```suggestion
FROM debian:12.5-slim
```